source/wp-content/themes/wporg-developer-2023/theme.json

```diff
@@ -158,7 +158,7 @@
 			"wporg-sidebar-container": {
 				"spacing": {
 					"margin": {
-						"top": "150px"
+						"top": "100px"
```

Comment:
Sorry, obviously more clarity required; on pages with breadcrumbs it needs to remain 150px, but for the top level handbook pages without breadcrumbs 100px.

The child page sidebars are now misaligned:

![Screenshot 2024-02-20 at 12 13 47 PM](https://github.com/WordPress/wporg-developer/assets/1017872/9e21f184-7e7f-49c8-a130-d7f96f9c5126)